### ISSUE TITLE
FRC-Control-System-Layout.svg has hard to read text

### ISSUE THREAD

**Issue by TheTripleV** [wpilibsuite/Control-System-Vectors#3]
The text in this image looks like it has been over-compressed and is hard to read, especially when zoomed out.

Link to github file: https://github.com/wpilibsuite/Control-System-Vectors/blob/master/layout-diagram/FRC-Control-System-Layout.svg
Link to image: https://raw.githubusercontent.com/wpilibsuite/Control-System-Vectors/master/layout-diagram/FRC-Control-System-Layout.svg

**Comment by Daltz333** [comment]
I don't believe it's a compression issue, I believe it's a font choice. There is no compression applied to most of the SVG as it's a vector graphic. We had a previous better font, but it had issues rendering due to it not being web safe.

**Comment by benjamin-antupit** [comment]
@TheTripleV, you're correct, something is up with the text in this file. It looks like some of the text outlines were converted to vector shapes and then simplified, which reduced readability. I think I can fix this.

Example of issue:
![image](https://user-images.githubusercontent.com/15172054/133272047-a4449169-f051-4e39-a235-cb51a088291d.png)
